docs/data-ai/data/query.md

```diff
@@ -51,7 +51,7 @@ Then, select either **SQL** or **MQL** from the **Query mode** dropdown menu on
 {{% tablestep number=2 %}}
 **Run your query**
 
-This example query returns 5 readings from a component called `my-sensor`:
+This example query returns 5 readings from all machines in your current organization where the component is called `my-sensor`:
```

Comment:
Still sounds confusing--maybe this? Not sure if it's the most recent 5 or not, but either way I think saying from any components called this makes more sense.
```suggestion
This example query returns the last 5 readings from any components named `my-sensor` in your organization:
```